mud-contracts/world-v2/package.json

```diff
@@ -17,24 +17,33 @@
     "test": "tsc --noEmit && mud test --forgeOptions=-vvv"
   },
   "dependencies": {
-    "@latticexyz/cli": "2.2.11",
-    "@latticexyz/schema-type": "2.2.11",
-    "@latticexyz/store": "2.2.11",
-    "@latticexyz/world": "2.2.11",
-    "@latticexyz/world-modules": "2.2.11",
-    "@latticexyz/gas-report": "2.2.11",
+    "@latticexyz/cli": "link:/Users/kooshaba/workspace/mud/packages/cli",
```

Comment:
should we publish this as a internal npm package until the MUD has this as part of the default ?

cc: @tinkrtailor

Reply:
i think we should wait for the official release, unless we have a good flow for internal packages